extended/pom.xml

```diff
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
note, this change happened when maven rewrote the new pom file, I suggest we keep it to eliminate future diffs.

Reply:
thanks! sgtm

as a follow-up, i'll do the openapi-generator migration and 1.14 api re-generation

/lgtm